### PR TITLE
US126098 - Add support for fr-on lang pack to HTML editor

### PR DESCRIPTION
**Pull request by grant-cleary**
Does what it says on the tin. :)

### CODE REVIEW

File: lang/fr-on.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/* eslint quotes: 0 */
```
Comment (grant-cleary) [inline lang/fr-on.js:1]: Copied from `fr.js` as per instructions.

**Comment by grant-cleary** [comment]
... Alright, I think at this point there just must be something wrong with the current goldens. This literally doesn't touch anything visual.

**Comment by dbatiste** [comment]
Do we need to do anything with [htmleditor.serge.json](https://github.com/BrightspaceUI/htmleditor/blob/master/htmleditor.serge.json)?

**Comment by dbatiste** [comment]
> ... Alright, I think at this point there just must be something wrong with the current goldens. This literally doesn't touch anything visual.

Yah seems like there's a timing issue.

**Comment by grant-cleary** [comment]
> Do we need to do anything with [htmleditor.serge.json](https://github.com/BrightspaceUI/htmleditor/blob/master/htmleditor.serge.json)?

Sending you a Slack link offline, but no, I don't believe so.

**Comment by grant-cleary** [comment]
As discussed offline, I'm disabling the visual diff tests for now while we look into these timing issues.

**Review by dbatiste — APPROVED** [review]

**Pull request merged**